components/gitpod-db/src/typeorm/migration/1683791172567-AddVerifiedFlagToOIDCClientConfig.ts

```diff
@@ -0,0 +1,27 @@
+/**
+ * Copyright (c) 2023 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License.AGPL.txt in the project root for license information.
+ */
+
+import { MigrationInterface, QueryRunner } from "typeorm";
+import { columnExists } from "./helper/helper";
+
+const table = "d_b_oidc_client_config";
+const column = "verified";
+
+export class AddVerifiedFlagToOIDCClientConfig1683791172567 implements MigrationInterface {
+    public async up(queryRunner: QueryRunner): Promise<void> {
+        if (!(await columnExists(queryRunner, table, column))) {
+            await queryRunner.query(
+                `ALTER TABLE ${table} ADD COLUMN ${column} tinyint(4) NOT NULL DEFAULT '0', ALGORITHM=INPLACE, LOCK=NONE`,
```

Comment:
I think we should also add a query that sets `verified=1` for any existing record where `active=1` so existing data stays consistent.